.sbtrc

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright 2022 ABSA Group Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Aliases in this file expected usage of test file naming conventions:
+#   - "UnitTest" suffix for test files and Suites which define unit tests
+#   - "IntegrationTest" suffix for test files and Suites which define integration tests
+
+# CPS QA types aliases
+# * Unit tests
+alias test=; testOnly *UnitTest
+
+# * Integration tests
+alias testIT=; testOnly *IntegrationTest
```

Comment:
**Idea:**
To be consistent, also plural:
```suggestion
alias testIT=; testOnly *IntegrationTests
```

Reply:
Addressed.